docs/reference.md

```diff
@@ -131,3 +131,62 @@ dedicated test-suite, consisting of several tests exercising various runtime
 behaviors, for each StableHLO Op. The tests can be found
 [here](https://github.com/openxla/stablehlo/tree/main/stablehlo/tests/) (e.g.
 interpret\_\*.mlir).
+
+### Testing guidelines
+
+**(G1) Do we need to test for all the supported types for every op?**
+
+We can use a combination of following rules to decide it:
```

Comment:
Indentation for this list doesn't seem to follow the recently approved guidelines. PTAL.

Reply:
Sure.